Classes/Controller/PageViewController.php

```diff
@@ -634,15 +635,17 @@ protected function getImage(int $page, MetsDocument $specificDoc = null): array
                 $files = $physicalStructureInfo['files'];
                 if (!empty($files[$fileGrpImages])) {
                     $file = $specificDoc->getFileInfo($files[$fileGrpImages]);
-                    $image['url'] = $file['location'];
-                    $image['mimetype'] = $file['mimeType'];
-
-                    // Only deliver static images via the internal PageViewProxy.
-                    // (For IIP and IIIF, the viewer needs to build and access a separate metadata URL, see `getMetadataURL` in `OLSources.js`.)
-                    if ($this->settings['useInternalProxy'] && !str_contains(strtolower($image['mimetype']), 'application')) {
-                        $this->configureProxyUrl($image['url']);
+                    if (Helper::filterFilesByMimeType($file, 'image', 'mimeType')) {
+                        $image['url'] = $file['location'];
+                        $image['mimetype'] = $file['mimeType'];
+
+                        // Only deliver static images via the internal PageViewProxy.
+                        // (For IIP and IIIF, the viewer needs to build and access a separate metadata URL, see `getMetadataURL` in `OLSources.js`.)
+                        if ($this->settings['useInternalProxy'] && !str_contains(strtolower($image['mimetype']), 'application')) {
```

Comment:
In line 638 you filter out anything, that hasn't an `image/*` MIME type, but in line 644 you still expect to find images with `application/*` MIME type? ;)

Reply:
You're Right. I actually missed that. 
I'll look over it when I'm back again.

Reply:
@sebastian-meyer I fixed the wrong inclusion of `configureProxyUrl` inside the if statement.